pkg/storage/system_intake_grb_reviewer.go

```diff
@@ -11,14 +11,15 @@ import (
 	"github.com/cms-enterprise/easi-app/pkg/appcontext"
 	"github.com/cms-enterprise/easi-app/pkg/models"
 	"github.com/cms-enterprise/easi-app/pkg/sqlqueries"
+	"github.com/cms-enterprise/easi-app/pkg/sqlutils"
 )
 
 // CreateSystemIntakeGRBReviewer creates a GRB Reviewer
-func (s *Store) CreateSystemIntakeGRBReviewer(ctx context.Context, tx *sqlx.Tx, reviewer *models.SystemIntakeGRBReviewer) error {
+func (s *Store) CreateSystemIntakeGRBReviewer(ctx context.Context, np sqlutils.NamedPreparer, reviewer *models.SystemIntakeGRBReviewer) error {
```

Comment:
Apparently multiple transactions will block error groups, so I changed this store method to accept the store or a tx via NamedPreparer.

Reply:
i am curious about this. is there an error?

Reply:
It would either block the thread and the operations would never finish or throw a duplicate key Postgres error.

Reply:
i guess i am more just curious if this would break something else, but i looked, and the only other place is using a `tx`, so it's probably fine